### DISCUSSION TITLE
Reply awaited

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#57] [General]
Dear Lars,

I have written to you privately regarding my sample-set and got no reply yet.

I need your feedback asap otherwise I am not able to send the set for your evaluation and beta testing.

Kindly check your mail.

Mark

**Comment by larspalo** [comment]
I've replied to your e-mail. I've been too busy lately...

**Reply by Bumblebee001** [reply]
I sent you a new link by email and had no reply from you whether you even received it or not. I assume you did't or my mail is passing off as junk.

Can you kindly reply?
